terraform/modules/elasticache-redis/main.tf

```diff
@@ -0,0 +1,41 @@
+# Pulled from https://github.com/turnerlabs/terraform-aws-elasticache-redis
+resource "aws_security_group" "redis" {
+  vpc_id = "${var.vpc_id}"
+
+  tags {
+    Name          = "${var.tag_name}"
+    environment   = "${var.tag_environment}"
+    team          = "${var.tag_team}"
+    application   = "${var.tag_application}"
+    contact-email = "${var.tag_contact-email}"
+    tag_customer  = "${var.tag_customer}"
+  }
+}
+
+resource "aws_elasticache_subnet_group" "default" {
+  name        = "subnet-group-${var.tag_team}-${var.tag_application}-${var.tag_environment}"
+  description = "Private subnets for the ElastiCache instances: ${var.tag_team} ${var.tag_application} ${var.tag_environment}"
+  subnet_ids  = ["${split(",", var.private_subnet_ids)}"]
+}
+
+resource "aws_elasticache_cluster" "redis" {
+  cluster_id           = "${var.cluster_id}"
+  engine               = "redis"
+  engine_version       = "${var.engine_version}"
+  maintenance_window   = "${var.maintenance_window}"
+  node_type            = "${var.instance_type}"
+  num_cache_nodes      = "1"
+  parameter_group_name = "${var.parameter_group_name}"
+  port                 = "6379"
+  subnet_group_name    = "${aws_elasticache_subnet_group.default.name}"
+  security_group_ids   = ["${aws_security_group.redis.id}"]
+
+  tags {
+    Name          = "${var.tag_name}"
```

Comment:
Can we name things consistently with the rest of the resources so we can identify them. Instead of `tvs2-staging` it would be good to have the resource in the name so: `tvs2-staging-redis`